pages/common/ghci.md

```diff
@@ -0,0 +1,24 @@
+# ghci
+
+> The Glasgow Haskell Compiler's interactive environment.
+> More information: <https://downloads.haskell.org/ghc/latest/docs/html/users_guide/ghci.html>.
+
+- Start a REPL (interactive shell):
+
+`ghci`
+
+- Start a REPL and load the specified Haskell source file:
+
+`ghci {{source_file.hs}}`
+
+- Start a REPL and enable a language option:
+
+`ghci -X{{language_option}}`
+
+- Start a REPL and enable almost all compiler warnings:
```

Comment:
`enabled` or `turned on` should be in past tense I think, because you start a REPL with an option turned on. Maybe:
```suggestion
- Start a REPL with compiler warnings that indicate potentially suspicious code:
```
Or
```suggestion
- Start a REPL with compiler warnings that indicate potentially suspicious code turned on:
```
Sorry, English is not my mother language cc @sbrl @owenvoke

Reply:
I wouldn't say "potentially suspicious" to be honest. Something about it sounds ...off. I changed the wording and the code-snippet a bit.

Reply:
I suggested that because that's the official language in the docs.

Reply:
Maybe my non-native ears are not qualified to rate the wording of english sentences then :D I will add the phrase.

Reply:
I'm from Argentina. The Oxford dictionary here is @sbrl :hand_over_mouth:.

Reply:
Lol! In this case, it really didn't need changing :P

It's fine as-is, so no need to adjust again, but in future it didn't need adjusting :laughing: